build.gradle.kts

```diff
@@ -51,7 +51,7 @@ allprojects {
 }
 
 group = "exchange.dydx.abacus"
-version = "1.7.31"
+version = "'1.7.31"
```

Comment:
@mike-dydx is this from the auto increment script or is this a remnant of our edits